image.md

```diff
@@ -0,0 +1,140 @@
+The `config.json` file contains the host independent configuration information necessary to execute the container.
+See [`config.json`](config.md) for more information.
+
+1. The root filesystem of the filesystem bundle.
+This directory MUST be present in the image, however it MAY be empty (not contain any files or folders).
+As specified in the `config.json` file, this directory contains the files that make up the root filesystem of the container.
+This directory MAY contain only a portion of the complete set of files that make up the filesystem bundle, in which case the remaining set of files MUST be found in one of the dependent images.
+
+1. References to dependent images.
+If present, this OPTIONAL information MUST be in a file called `required.json`.
+This file MUST be a json encoded array of references to images as specified in the [Dependent Images](#dependent-images) section.
+
+1. Dependent images.
+An image MAY choose to include dependent images as directories within the image.
+If present, each dependent image MUST be placed in a directory whose name is the same as its ID.
+Within each directory MUST be a complete OCI compliant image definition.
+Note that within each dependent image's directory there MAY be a whiteout.json file.
+
+1. Whiteout list.
+An image MAY choose to include a list of files that are to be considered to be deleted from the root filesystem even though they are included within the image's root filesystem directory.
+It is an implementation detail to decide how this whiteout list is processed but an implementation MUST ensure that any container that is created from this image will behave the same as if those files were not in the root filesystem at all.
+The whiteout list MUST be in a file called `whiteout.json`.
+See the [Whiteout List](#whiteout-list) section for more details.
+
+1. Extension files.
+An image MAY choose to include additional files in the image.
+
+1. The ID of the image.
+This REQUIRED information MUST be in a file called `ID`.
+See the [Image ID](#image-id) section for more details.
+
+## Serialization of an Image
+
+An image MUST be a `tar` of the pieces of data that make up an image as specified in the [Image Contents](#image-contents) section.
+The order in which the data is serialized in the `tar` MUST be the same as the order specified in that section.
+The serialization of the "dependent images" MUST be in the same order in which those dependent images are listed in the `requires.json` file.
+The serialization of the  "extension files" MUST be in case-sensitive alphabetical order.
+
+Each file or directory specified in the [Image Contents](#image-contents) section MUST appear at the root of the `tar` without a leading `/` in their names.
+
+For example, a listing of the `tar` of an image might look like this:
+```
+config.json
+rootfs/
+    bin/myapp
+requires.json
+sha512-1d62d181e9c1322d56ccd3a29d05018399147a16188dbd861c0279ad0dd7e14c/
+    config.json
```

Comment:
This dependent image doesn't have a rootfs, and is likely an example of [why I don't think we want to require one][1].

[1]: https://github.com/opencontainers/specs/pull/293/files#r48905528